README.md

```diff
@@ -1 +1,46 @@
-# tdd-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# tdd-project
+
+> In this project we used TDD to implement all the methods on the Solver class.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+In order to run locally this project type the following commands over the terminal in Linux or Mac or the Windows cmd console:
+
+- git clone https://github.com/piero-vic/tdd-project
+- cd tdd-project
+
+
+## Authors
+
+👤 **Amel Khiri**
+
+- GitHub: [@Algerina](https://github.com/Algerina)
+- LinkedIn: [Amel Khiri](https://linkedin.com/in/amel-khiri-qahwadji-37a550135)
+
+👤 **Eapen Zacharias**
+
+- GitHub: [@eapenzacharias](https://github.com/eapenzacharias)
+- Twitter: [@eapenzac](https://twitter.com/eapenzac)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eapenzac)
+
+👤 **Piero Lescano**
+
+- GitHub: [@piero-vic](https://github.com/piero-vic)
+- Twitter: [@v1ccenzo](https://twitter.com/v1ccenzo)
+- LinkedIn: [piero-lescano](https://linkedin.com/in/piero-lescano)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/piero-vic/tdd-project/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] All tests are passing. However, Let's add the `test` section to the README and include instructions on how to run your tests successfully. Please include instructions like `To run the tests, make sure you install RSpec by running this command **gem install rspec** and after installation, run **rspec spec/solver_spec.rb ** to see all the test passing`